client/layout/logged-out.jsx

```diff
@@ -10,33 +10,88 @@ import { connect } from 'react-redux';
  * Internal dependencies
  */
 import MasterbarLoggedOut from 'layout/masterbar/logged-out';
+import { getOAuth2ClientData, showOAuth2Layout } from 'state/login/selectors';
 import { getSection } from 'state/ui/selectors';
 import OauthClientLayout from 'layout/oauth-client';
-import { showOAuth2Layout } from 'state/login/selectors';
+
+const clients = {
```

Comment:
Do you think we'll need this information outside of the `OAuthClientLayout` component?